internal/eval/eval_test.go

```diff
@@ -5733,3 +5734,40 @@ func BenchmarkEvalJSONSTRAPPEND(b *testing.B) {
 		evalJSONSTRAPPEND([]string{"doc1", "$..a", "\"bar\""}, store)
 	}
 }
+
+func testEvalLINSERT(t *testing.T, store *dstore.Store) {
```

Comment:
These tests should also validate the final state of the list (wherever applicable).

Reply:
Done.